ci_scripts/shared-prelude-context/install-rust.sh

```diff
@@ -1,6 +1,5 @@
 #/bin/bash
 set -xe
 
-apt -y install curl
 curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly-2019-03-23 -y
```

Comment:
the nightly version seems like something that may want to be controlled.  maybe put it in a variable, maybe even overwritable by the caller?  might also be premature optimisation

Reply:
I would say premature optimisation, sure we can have a variable for it but i think it's clear enough that one liner.